test/suite/test_compact11.py

```diff
@@ -31,8 +31,8 @@
 from wtbackup import backup_base
 
 # test_compact11.py
-# Verify background compaction and incremental backup behaviour. The block modifications bits in an
-# incremental backup should never be cleared.
+# Verify background compaction and backup behaviour. The block modifications bits in an
+# backup should never be cleared when background compact is working on tables.
```

Comment:
Why did you remove "incremental" here at the beginning of the line? The block mod bits are specifically only for incremental backup. (And it leaves the comment with '...in an backup' which is wrong grammatically.)